apps/jupyter-lab/template/script.sh.erb

```diff
@@ -7,8 +7,13 @@ echo "TIMING - Starting main script at: $(date)"
 cd "$OOD_WD" || true
 
 TOOLCHAIN=<%= context.toolchain %>
-LOAD_SCIPY=<%= context.load_scipy %>
-LOAD_MATPLOTLIB=<%= context.load_matplotlib %>
+
+MODULES_DIR="/apps/brussel/${VSC_OS_LOCAL}/${VSC_ARCH_LOCAL}${VSC_ARCH_SUFFIX:-}/modules"
+MODULES_TOOLCHAIN="${MODULES_DIR}/${TOOLCHAIN}/all"
+MODULES_SYSTEM="${MODULES_DIR}/system/all"
+export MODULEPATH="${MODULES_TOOLCHAIN}:${MODULES_SYSTEM}"
+echo "MODULEPATH set to: ${MODULEPATH}"
```

Comment:
Can you add a comment on the reason for this? I guess to only show compatible modules?